public/data/CBID2.json

```
@@ -0,0 +1,28 @@
+{
+	"name": "Christine Belzie",
+	"type": "personal",
+	"bio": "An Instructional Designer who codes",
+	"avatar": "https://github.com/CBID2.png",
+	"links": [
+	  {
+		"name": "Follow me on GitHub",
+		"url": "https://github.com/CBID2",
+		"icon": "github"
+	  },
+	  {
+		"name": "Follow me on Linkedin",
```

I recommend capitalizing the `i` here. What do you think? 🙂
```suggestion
		"name": "Follow me on LinkedIn",
```